### PR TITLE
Display WordPress Posts Widget: Add new option to open links in a new tab

### PR DESCRIPTION
**Pull request by mikeyarce**
34c9dfaa-afb4-11e4-8ed9-3682f0b265cd

### CODE REVIEW

**Comment by mikeyarce** [comment]
This fixes the problems with my bad PR #1603

File: modules/widgets/wordpress-post-widget.php [hunk 1]
```diff
@@ -161,6 +165,11 @@ public function form( $instance ) {
 			$number_of_posts = 5;
 		}
 
+		$open_in_new_window = false;
+		if ( isset( $instance[ 'open_in_new_window'] ) ) {
+		    $open_in_new_window = $instance[ 'open_in_new_window'];
```
Comment (samhotchkiss) [inline modules/widgets/wordpress-post-widget.php]: You've got an extra space there inside the [

**Comment by samhotchkiss** [comment]
Other than a very minor style nitpick, this looks good, @mikeyarce !

**Comment by dereksmart** [comment]
nice job, will merge as soon as whitespace is fixed.

**Comment by dereksmart** [comment]
whitespace clean, good to go!

**Pull request merged**